### PR TITLE
Styling updated dashboard

### PR DESCRIPTION
**Pull request by anniethewebcoder**
styling the updated dashboard (#60)

I styled the login and sign up components. Now you can switch between two components instead of stacking.

### CODE REVIEW

File: src/components/AccountCreation/AccountCreationContainer.tsx
Comment (marice-romero) [inline src/components/AccountCreation/AccountCreationContainer.tsx:1]: Really neat idea to conditionally render the login form, makes everything look cleaner.

**Review by marice-romero — APPROVED** [review]
Great work, looks so pretty and professional!

File: src/components/AccountCreation/AccountCreationContainer.tsx
Comment (anniethewebcoder) [reply src/components/AccountCreation/AccountCreationContainer.tsx:1]: Thank you. :) It took me hours to figure it out without having to make drastic changes. I had to re-read a couple of chapters in Road to React.

**Review by catbus00 — APPROVED** [review]
Great work!

**Comment by dargaCode** [comment]
What a cool idea, thanks @anniethewebcoder 🙏

**Pull request merged**